### PR TITLE
increase emergency tank w_class

### PR DESCRIPTION
**Pull request by hornygranny**
in 2011 this was accidentally-on-purpose changed during the DANGERCON meme to allow e-tanks to fit in pockets, which is the opposite of danger and makes hull breaches not really a problem

:cl: 
tweak: Emergency tanks no longer fit in pockets
/:cl:

### CODE REVIEW

**Comment by coiax** [comment]
I thought it was going to be "larger capacity", so I was ready with muh hugbox memes. But this is fine. More danger.

**Comment by Zilenan91** [comment]
I was really surprised when an oxygen tank fit in your pockets :+1:

**Comment by KorPhaeron** [comment]
This will probably mildly irritate sec and ops and otherwise not matter

**Comment by YakumoChen** [comment]
wait, what's the point of those tiny tanks then if they don't fit in any decent slot?

**Comment by GunHog** [comment]
It means you will have to sacrifice your belt slot or suit storage slot (if your suit slot item takes tanks) if you want to use it. In a pinch, you may be forced to hold it in your hand, but that is basically death if you get stunned in any fashion.

**Comment by coiax** [comment]
@GunHog HG is going through a necroPR phase

**Comment by Niknakflak** [comment]
I will be upset if this is merged

**Comment by bobdobbington** [comment]
If you merge this PR, you can upset Niknakflak

**Comment by Fox-McCloud** [comment]
Gross, no. 

:-1:  It's really not that big of a deal to use it in your pocket.

**Comment by Core0verload** [comment]
Another pointless nerf? Down in the downvotes hell it goes. :-1:

**Comment by WJohn** [comment]
I'm indifferent to this.

**Comment by Firecage** [comment]
Now, look. Sure, this might make sense if we wanted to add more realism to the game, but from a gameplay point of view? Fuck no, horrible change, and why did this even get test merged?

**Comment by coiax** [comment]
What if you make an even smaller tank that goes in the internals box that is pocket sized; call it a pocket emergency oxygen tank. Maybe around a half litre?

**Comment by Davdichan** [comment]
This would make them entirely useless, you'd be better off getting a larger tank and putting that on your backslot or exosuit slot, the e-tanks don't have anywhere near the capacity as the larger tanks and making them eat the belt slot is just gross.  Oxyloss isn't really that threatening to the crew anyways, it's pressure damage that makes breaches deadly. 

You'll see hardsuited or firefighter suited people fixing breaches, but you'll never see someone with just internals sticking around to fix a breach unless it's absolutely vital to seal it.

**Comment by coiax** [comment]
I think the point is they're supposed to be for emergencies.

**Comment by Deputi** [comment]
"The emergency oxygen tank is too big for the box!"

BUT HOW DID THEY EVEN FIT IT IN THE BOX IN THE FIRST PLACE?

**Comment by GunHog** [comment]
I wonder how the Syndicate fits a freaking SPACE SUIT in a tiny box...

**Comment by PKPenguin321** [comment]
they fold it really really well then vacuum seal it

**Comment by PKPenguin321** [comment]
oh and also i played this ingame and it's really bad

**Comment by Isratosh** [comment]
I am absolutely against this pull request.

**Comment by optimumtact** [comment]
Well hmm, I'm afraid that it's been too long that this power creep has become the accepted norm, a more acceptable alternatives will be changing the maximum amount of oxygen or air in a pocket tank to make it less supposedly powerful.

**Comment by KorPhaeron** [comment]
It's been like 5 years now yeah. I don't know if it's even fair to call it creep at this point.

And keep in mind capacity was also nerfed when they were made small, and pressure damage was added so hull breaches are dangerous even if you have internals on. Also in that update: Temperature actually hurting you, vs only doing damage on a space tile.

As far as "power" goes, I'd take the old "lasts the entire round breach immunity" on your belt vs "keeps you from passing out but you'll die anyway" that fits in your pocket.

tl;dr this is cherrypicking a very small change out of a larger package erro did, and calling it power creep in a vacuum (no pun intended)

**Comment by kilog** [comment]
:ok_hand:
